### PR TITLE
Add Coffeescript, Scala, FSharp, TCL and Nim icons and add SQL as "storage"

### PR DESCRIPTION
**Pull request by Hugoonreplit**
I would like to add these file icons all from the source svgrepo.com and with a size of 14x14. Also I've modified file_types.json in order to add the file types and path to the image aswell as added SQL as a storage type so it's linked to an icon.

Here is how these new changes would look like:
Captura de pantalla 2024-02-26 a las 19 30 33

Release Notes:

- Added icons for Coffeescript, F#, Nim, Scala, and TCL files.
- Updated icon for SQL files.

### CODE REVIEW

File: assets/icons/file_icons/fsharp.svg
[the remark this answers is not in the document]
Comment (Hugoonreplit) [reply assets/icons/file_icons/fsharp.svg:1]: Thank you for spotting this out, I've updayed the logo, does that look suitable?

[the remark this answers is not in the document]
Comment (Hugoonreplit) [reply assets/icons/file_icons/fsharp.svg:1]: Thank you for spotting this out @maxdeviant. I've updated the logo. There is different type of F# logo's, but I think this is the most suitable one

[the remark this answers is not in the document]
Comment (Hugoonreplit) [reply assets/icons/file_icons/fsharp.svg:1]: @maxdeviant Is this ready to be merged, or should I make any other changes

**Comment by maxdeviant** [comment]
Thanks, they look good!

Screenshot 2024-02-28 at 11 07 49 AM

**Pull request merged**